crates/hyperswitch_domain_models/src/merchant_connector_account.rs

```diff
@@ -136,6 +119,68 @@ impl MerchantConnectorAccount {
     }
 }
 
+#[cfg(feature = "v2")]
+/// Holds the payment methods enabled for a connector along with the connector name
+/// This struct is a flattened representation of the payment methods enabled for a connector
+#[derive(Debug)]
+pub struct PaymentMethodsEnabledForConnector {
+    pub payment_methods_enabled: common_types::payment_methods::RequestPaymentMethodTypes,
+    pub payment_method: common_enums::PaymentMethod,
+    pub connector: String,
```

Comment:
Should the `connector` be an Enum instead?

Reply:
This will be a change that will affect v1 as well, I have already looked into this and will take up as necessary